tyrant/cogs/magic_8_ball.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import random
+
+from discord.ext import commands
+from discord.ext.commands import Bot, Cog, Context
+
+from tyrant import constants
+
+log = logging.getLogger(__name__)
+
+
+class Magic8Ball(Cog):
+    """Provide Answers for generic yes or no questions"""
```

Comment:
```suggestion
    """Provide answers for generic yes or no questions."""
```
Imperfect docstrings? NEVER! 🍋